Identity/Identity.DataAccess/Models/Entities/UserEntity.cs

```diff
@@ -10,16 +10,16 @@ public class UserEntity
 	public int Id { get; set; }
 
 	[Required]
-	public string Username { get; set; }
+	public required string Username { get; set; }
 
-	[Required]
-	public string PasswordHash { get; set; }
+    [Required]
```

Comment:
Some weird formatting mismatch happened

Reply:
Strange, it shows up on mobile but is not present in VS or GitHub on desktop?
![image](https://github.com/BenasB/vu-psk/assets/95697934/81b1326b-08d6-4617-88d1-e0326b6a5a24)

Reply:
Got it. Perhaps someone else will format it with their changes if that shows up in their IDE.